### DISCUSSION TITLE
Fehlermeldung nach Update zweiter versuch

### DISCUSSION THREAD

**Discussion by heliuli** [community/community#129891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hallo ,
Entschuldigt bitte dass mein Englisch schon lange her ist.

wie beschrieben nach Update läuft evcc nun nicht mehr .

Der Empfänger dieser zeilen sollte diese an die Stelle bringen wo mir bitte in Deutsch geholfen wird.
Vielleicht kann mir "Volker "  helfen dass  hat in der vergangenheit in deutsch geklappt
Es gibt anscheinend ein Problem mit der Konfigurationsdatei.
Helft mir, wie ich diese Anpassen muss ( Bitte in Deutsch).

 Gruss Marco

**Comment by ebndev** [comment]
Hi @heliuli, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
